README.md

```diff
@@ -6,22 +6,36 @@ This repository contains resources for deploying the Deep Log Inspection system
 
 This project is part of [FIWARE][1]. Credits to [monasca-docker][2] and [docker-elk][3].
 
-Quick Start
------------
+## Quick Start
 
-To get the Deep Log Inspection system running, you can use [Docker Engine][4] in [Swarm mode][5]. In order for Elasticsearch to boot correctly, user limits must be set via [Docker configuration][7].
+Installing and running the Deep Log Inspection system requires [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Docker Engine](https://www.docker.com/) in [Swarm mode](https://docs.docker.com/engine/swarm/):
 
-First, create the overlay network where the services will be deployed, and the storage volumes for Elasticsearch data and logs:
+1. Set up the [Docker Swarm cluster](https://docs.docker.com/get-started/part4/#create-a-cluster) where services will be deployed
+2. Download the project by cloning the git repository:
 
-    docker network create -d overlay backend
-    docker volume create esdata
-    docker volume create eslogs
+        git clone https://github.com/martel-innovate/deep-log-inspection.git
 
-Let `deeplog` be the name of the stack where we deploy all services to, then:
+3. Set user limits in Docker [configuration file](https://github.com/martel-innovate/deep-log-inspection/blob/master/log-server/config/docker.service). For more details, refer to the [Docker configuration section](doc/install/docker.md)
+4. Create the overlay network where the services will be deployed, and the storage volumes for Elasticsearch data and logs:
 
-    docker stack deploy -c docker-compose.yml deeplog
+        docker network create -d overlay backend
+        docker volume create esdata
+        docker volume create eslogs
 
-The system is meant to be deployed on a multi-node cluster, in which the master node is accessible with three valid domain names. Let `example.com` be the main domain name and `elastic.example.com` and `kibana.example.com` two alternative domains (in this page and throughout the full guide). Then, assuming all goes well, the following services should be exposed on the master node:
+5. Let `deeplog` be the name of the stack where we deploy all services to, then:
+
+        docker stack deploy -c docker-compose.yml deeplog
+
+The compose file can be found [here](https://github.com/martel-innovate/deep-log-inspection/blob/master/log-server/docker-compose.yml).
+
+If needed, the environment can be completely cleaned:
+
+    docker stack rm deeplog
+
+For the full CLI reference, please refer to the [Docker CLI reference guide](https://docs.docker.com/engine/reference/commandline/cli/).
+
+## Services and DNS
+The system is meant to be deployed on a multi-node cluster, in which the master node is accessible with three valid domain names. Let `example.com` be the main domain name and `elastic.example.com` and `kibana.example.com` two alternative domains (in this page and throughout the full guide). Then, the following services should be exposed on the master node:
```

Comment:
`elastic.example.com` and `kibana.example.com` give that are mapped to an ip, are name servers (and eventually subdomains)

Reply:
example.com, elastic.example.com and kibana.example.com are all mapped to the same IP, so I guess it's a name server with two alternative names?